internal/cli/inputs.go

```diff
@@ -17,13 +17,13 @@ import (
 const (
 	flagApp      = "app"
 	flagAppShort = "a"
-	flagAppUsage = "the remote Realm app name or id"
+	flagAppUsage = "Specify the name or ID of a Realm app"
```

Comment:
[q] Should we mention Client App ID vs the Realm App ID?

Reply:
Nowhere in the CLI is the App ID (as you know the, the ObjectID) really a part of the user experience.  So for brevity, we generally (and this is existing behavior) refer to what we call the client app id as just the app id (notice how we rename it `"app_id"` when exporting out the `realm_config.json`/`config.json`/etc.

Going to leave this as is, until user feedback indicates we need to do otherwise

Reply:
Yeah, just didn't want to introduce confusion with using the term `ID` also for a `MongoDB Atlas project`, which is the `ObjectID`.